src/Acme/UserBundle/Entity/User.php~

```diff
@@ -0,0 +1,33 @@
+<?php
```

Comment:
I ended up using VS Code to edit my code. I was jumping a little between computers since my Windows partition crashed on my own laptop, so I had to improvise. I honestly didn't notice the backups remained, thanks! I shall update the .gitignore